test/test.js

```diff
@@ -828,8 +827,9 @@ function startBrowsers(rootUrl, initSessionCallback) {
       `&delay=${options.statsDelay}` +
       `&masterMode=${options.masterMode}`;
     const startUrl = rootUrl + queryParameters;
+    const headless = options.strictVerify;
```

Comment:
*Probably bike-shedding, but anyway:* Would it make *slightly* more sense to use e.g. the `noPrompts` option here since that's "feels" somewhat easier to logically related to a no-UI mode!?